lib/fluent/plugin/out_fork.rb

```diff
@@ -62,9 +63,11 @@ def emit(tag, es, chain)
           end
         end
 
-        values.reject{ |value| value.to_s == '' }.each do |value|
+        values.reject{ |value| value.to_s == '' }.each_with_index do |value, i|
```

Comment:
I think counting without empty value is better and we should not care the log contains empty values which is not supposed to be there.